src/App/App.tsx

```diff
@@ -69,38 +91,84 @@ class App extends Component<{}, AppState> {
     });
   }
 
+  async captureAvatarImage() {
+    const node = document.getElementById(this.state.avatarId);
+    if (node) {
+      const dataUrl = await domtoimage.toPng(node);
+      console.log(dataUrl);
+      this.setState({ avatarImageDataUrl: dataUrl }, () => {
+        console.log("State updated:", this.state.avatarImageDataUrl);
+      });
+    } else console.log("node is not exist");
+  }
+
+  handleFormSubmit(
+    formData: { name: string; region: string; role: string },
+    imageDataUrl: string
+  ) {
+    console.log("Form submitted", formData, imageDataUrl);
+  }
+
   render() {
-    const { config, shape } = this.state;
+    const { config, shape, avatarImageDataUrl } = this.state;
     return (
-      <div className="App flex flex-col min-h-screen">
-        <Header title="Avatar Generator" />
+      <div className="App flex flex-col min-h-screen overflow-x-hidden">
+        <Header title="AVANART" />
         <main className="flex-1 flex flex-col items-center justify-center">
-          <div id={this.state.avatarId} className="mb-10">
-            <ReactNiceAvatar
-              className="w-64 h-64 highres:w-80 highres:h-80"
-              hairColorRandom
-              shape={this.state.shape}
-              {...config}
+          <Routes>
+            <Route path="/about-us" element={<AboutUs />} />
+            <Route
+              path="/"
+              element={
+                <>
+                  <div id={this.state.avatarId} className="mb-10">
+                    <ReactNiceAvatar
+                      className="w-64 h-64 highres:w-80 highres:h-80"
+                      hairColorRandom
+                      shape={this.state.shape}
```

Comment:
On line 113 you did destructuring, so you could directly access all of it without "this.state"

Reply:
If you are talking about <app/> component, this structure comes from react nice avatar and we didn't change it, but I worked on form page and tried to stick role and region on top of image after clicking the submit btn. 
#33

Reply:
The Form looks all right to me, thank you Shadi